### ISSUE TITLE
Grand Theft Auto V (GTA V) problems

### ISSUE THREAD

**Issue by isiddharthasharma** [Heroic-Games-Launcher/HeroicGamesLauncher#1069]
Device MacOS:

Error - Command failed: "/Applications/Heroic.app/Contents/Resources/app.asar.unpacked/build/bin/darwin/legendary" launch 9d2d0eb64d5c44529cece33fe2a46482    
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Checking for updates...
[cli] CRITICAL: Metadata for "9d2d0eb64d5c44529cece33fe2a46482" does not exist, cannot launch!

**Comment by flavioislima** [comment]
This is related with #1067 same cause.

**Comment by isiddharthasharma** [comment]
Note: I fixed this GTA V issue by going to old version 1.11.0-alpha and it is working fine for GTA V.

**Comment by flavioislima** [comment]
I dont think you need to go that far, should work fine on 2.2.1.

**Comment by isiddharthasharma** [comment]
I tried Oden Hotfix(2.2.1) and downloading GTA V, it got stuck at downloading at 90-95% then all of sudden reset to 0. Then I downloaded again and it gave same problem. Then I used that old version and downloaded full game and made copy in my hard drive.

Also verifying file integrity didn't work for me on 2.2.1 too which worked good on that older version.

I'm mentioned a lot of that older version because most of things are working fine for me at that version and also Andrew Tsai made video using that version only
